routes/resource.js

```diff
@@ -7,22 +7,37 @@ module.exports = (db) => {
     fetch(process.env.API_URL + "/categories")
       .then((data) => data.json())
       .then((data) => {
-        console.log(data);
-        res.render("resource/new", { categories: data, user: req.session.userId });
+        res.render("resource/new", {
+          categories: data,
+          user: req.session.userId,
+        });
       })
       .catch((err) => {
         res.redirect("/");
       });
   });
 
   router.get("/:id", (req, res) => {
+    const loggedInUser = req.session.userId;
     fetch(process.env.API_URL + "/resource/" + req.params.id)
       .then((data) => data.json())
       .then((json) => {
         if (json) {
-          console.log("INDEX", json);
-          res.render("resource", { resource: json[0], user: req.session.userId });
+          const resourceData = json[0];
+          db.query(
+            `SELECT *, username FROM resource_comments JOIN users ON user_id = users.id WHERE resource_id = $1 ORDER BY resource_comments.updated_at ASC;`,
+            [req.params.id]
+          ).then((data) => {
+            const commentData = data.rows;
+            const templateVars = {
+              resource: resourceData,
+              comment: commentData,
+              loggedInUser: loggedInUser,
+            };
```

Comment:
this logic could be performed in the get controller for `/api/resource/:id` so that we aren't hitting the API for some of the resource data, and then directly querying the database for the resource comments data.